src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: 0 auto;
+  padding: 10px;
+  width: $calendar-width;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  gap: $gap-value;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $side;
+    width: $side;
+    border: $border-value;
+    background-color: $bg-color;
+    box-sizing: border-box;
+    transition: translate 0.5s, background-color 0.5s;
```

Comment:
your transition isn't working, you should add it to transform property, not translate